src/App.css

```diff
@@ -1,3 +1,24 @@
-.App {
-  text-align: center;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  padding: 4%;
+  background-color: #f5f6fa;
+}
+
+.center {
+  margin: 0 auto;
+  width: 95%;
+  box-shadow: 10px 10px 20px 0 rgba(0, 0, 0, 0.3);
+}
+
+.container {
+  margin: 0 6rem 0 6rem;
+  min-height: 78vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
 }
```

Comment:
- [ ] As in the project requirement `Your final design should match Zeplin's design (this includes all elements + the same styling)` So I am leaving a comment below please fix color, font etc by seeing your design and original design.
### Your design
![his](https://github.com/hamzaw9/book-store/assets/81727988/0bfe577e-7998-4331-9d24-0ebf328b73da)

### Original design
![original](https://github.com/hamzaw9/book-store/assets/81727988/db64619a-4820-4c09-bb6d-e5cfad262084)